test/spec/modules/docereeAdManagerBidAdapter_spec.js

```diff
@@ -25,6 +25,7 @@ describe('docereeadmanager', function () {
           userid: '',
           zipcode: '',
           userconsent: '',
+          platformUid: ''
         },
```

Comment:
As discussed in the last submission, this needs some improvement. Also could you discuss why you're paying full names in the clear? This is a quite unusual practice

Reply:
Consent to Process the Consumer’s Sensitive Data Consisting of Genetic Data for the Purpose of Uniquely Identifying an Individual / Natural Person requires positive opt in. You should check gpp string before processing some of these parameters.

References:

Cal. Civ. Code 1798.100(a)(2), 1798.121(a), and 1798.135(a)
Virginia Code 59.1-578(A)(5)
Colo. Rev. Stat. 6-1-1308(7)
Utah Code 13-61-302(3)(a)
Conn. PA 22-15, Sec. 6(a)(4)

Reply:
no changes will be merged without appropriate test coverage

Reply:
Hi; I talked  with other prebid leadership, we decided to merge your pr with a few tweaks